apps/api/prisma/migrations/20221018101250_create_folders_table/migration.sql

```diff
@@ -0,0 +1,32 @@
+BEGIN TRY
+
+BEGIN TRAN;
+
+-- CreateTable
+CREATE TABLE [dbo].[Folder] (
+    [id] INT NOT NULL IDENTITY(1,1),
+    [displayNameEn] NVARCHAR(1000) NOT NULL,
```

Comment:
no welsh?

Reply:
At this point, I am not sure when the requirements are going to come for Welsh. So i will leave a space for it, but not worry about it